VAMobile/src/components/CommonErrorComponents/ErrorComponent.tsx

```diff
@@ -29,13 +33,55 @@ const ErrorComponent: FC<ErrorComponentProps> = (props) => {
   const features = ScreenIDToDowntimeFeatures[props.screenID]
   const isInDowntime = oneOfFeaturesInDowntime(features, downtimeWindowsByFeature)
 
-  const getSpecificErrorComponent: FC<ErrorComponentProps> = ({ onTryAgain, screenID }) => {
+  const getSpecificErrorComponent: FC<ErrorComponentProps> = ({ onTryAgain, screenID, reactQueryError }) => {
     const tryAgain = onTryAgain ? onTryAgain : storeTryAgain
     const errorType = errorsByScreenID[screenID] || ''
 
     if (isInDowntime) {
       return <DowntimeError screenID={screenID} />
     }
+
+    if (reactQueryError) {
+      if (isErrorObject(reactQueryError)) {
```

Comment:
Since these `if` statements are the only clauses, they can be combined into a single `if` statement

Reply:
updated